go.mod

```diff
@@ -69,6 +69,8 @@ require (
 	github.com/libp2p/go-ws-transport v0.1.0
 	github.com/mattn/go-isatty v0.0.9 // indirect
 	github.com/miekg/dns v1.1.16 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
I didn't see any new imports, so I am guessing you may need to run `go mod tidy`